Lecture3/Task1/utils/IOFileReader.java

```diff
@@ -0,0 +1,36 @@
+package utils;
+
+import java.io.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class IOFileReader {
+
+	public static void getFileInfo() {
+		System.out.println("Info Written to File:");
+		File listOfProducts = new File("M://GitHub/products.txt");
+		try {
+			BufferedReader getInfo = new BufferedReader(new FileReader(listOfProducts));
+			String productInfo = getInfo.readLine();
+			while (productInfo != null) {
```

Comment:
throw some exception in case if `productInfo == null`

Reply:
done